cmake/ConfigSGX.cmake

```
@@ -10,27 +10,27 @@ execute_process(COMMAND getconf LONG_BIT OUTPUT_VARIABLE VAR_LONG_BIT)
 # set them to the usual defaults.
```

needs to change as it's failing instead of defaulting.

---

missed... fixed!